guide/src/StyleGuide.js

```diff
@@ -141,7 +141,7 @@ export default React.createClass({
             main: {
                 background: "whitesmoke",
                 width: "100%",
-                margin: "50px 0px 50px 250px", 
+                margin: "50px 0px 0px 250px", 
```

Comment:
The value "50px auto 0px" will fix the bottom and set the right and left back to "auto" how it was before last commit. 

Don't want to nitpick just setting a standard for PR especially once interns are also contibuting